speed-dreams/source-2.2.3/data/data/menu/EndExperimentMenu.xml

```diff
@@ -5,27 +5,28 @@
 <params name="endexperiment">
     <section name="dynamic controls">
 
-        <section name="ok">
-            <attstr name="type" val="text button"/>
-            <attstr name="show box" val="no"/>
-            <attstr name="text" val="OK"/>
-            <attstr name="tip" val="Exit from the game"/>
-            <attnum name="x" val="260"/>
-            <attnum name="y" val="290"/>
-            <attnum name="width" val="320"/>
-            <attstr name="h align" val="left"/>
-            <attstr name="font" val="big"/>
-            <attstr name="color" val="0xFFFFFF"/>
-            <attstr name="focused color" val="0xFFFFFF"/>
-            <attstr name="pushed color" val="0xFFFFFF"/>
-            <attnum name="image x" val="-48"/>
-            <attnum name="image y" val="0"/>
-            <attnum name="image width" val="320"/>
-            <attnum name="image height" val="31"/>
-            <attstr name="focused image" val="data/img/bg-button-big-focused.png"/>
-            <attstr name="enabled image" val="data/img/bg-button-big.png"/>
-            <attstr name="pushed image" val="data/img/bg-button-big-pushed.png"/>
-        </section>
+		<!-- OK button -->
+		<section name="ok">
+			<attstr name="type" val="text button"/>
+			<attstr name="show box" val="no"/>
+			<attstr name="text" val="Save the data"/>
+			<attstr name="tip" val="Exit from the experiment"/>
+			<attstr name="h align" val="left"/>
+			<attnum name="x" val="440"/>
+			<attnum name="y" val="41"/>
+			<attnum name="width" val="120"/>
+			<attstr name="font" val="medium"/>
+			<attstr name="color" val="0xFFFFFF"/>
+			<attstr name="focused color" val="0xFFFFFF"/>
+			<attstr name="pushed color" val="0xFFFFFF"/>
+			<attnum name="image x" val="-80"/>
+			<attnum name="image y" val="-9"/>
+			<attnum name="image width" val="300"/>
+			<attnum name="image height" val="40"/>
+			<attstr name="focused image" val="data/img/bg-button-big-focused.png"/>
+			<attstr name="enabled image" val="data/img/bg-button-big.png"/>
+			<attstr name="pushed image" val="data/img/bg-button-big-pushed.png"/>
+		</section>
```

Comment:
I think you might still have tabs as default in your IDE, replace tabs with spaces.